docs/rules/object-shorthand.md

```diff
@@ -152,6 +154,54 @@ var foo = {
 };
 ```
 
+When set to `"consistent"` the following will warn:
+
+```js
+/*eslint object-shorthand: [2, "consistent"]*/
+/*eslint-env es6*/
+
+var foo = {
+    a,
+    b: 'foo',
+};
+```
+
+The following will *not* warn:
+
+```js
+/*eslint object-shorthand: [2, "consistent"]*/
+/*eslint-env es6*/
+
+var foo = {
+    a: a,
+    b: 'foo'
+};
+```
+
+The following will also *not* warn:
```

Comment:
You can combine this with the preceding example. No need to have two code blocks.